app/controllers/api/pdf_forms_controller.rb

```diff
@@ -0,0 +1,110 @@
+class Api::PdfFormsController < Api::ApplicationController
+  def create
+    @pdf_form = PdfForm.new(pdf_form_params.merge(user_id: current_user.id))
+
+    authorize @pdf_form
+
+    if @pdf_form.save
+      render json: @pdf_form, status: :created
+    else
+      render_error "pdf_form.create_error",
+                   {
+                     message_opts: {
+                       error_message: @pdf_form.errors.full_messages.join(", ")
+                     },
+                     status: 422
+                   }
+    end
+  end
+
+  def index
+    @pdf_forms = policy_scope(PdfForm)
+    render_success @pdf_forms, nil, { blueprint: PdfFormBlueprint }
+  end
+
+  def generate_pdf
+    @pdf_form = PdfForm.find(params[:id])
```

Comment:
oftentimes a before_action is used for this

like
before_action :set_pdf_form, only: .....